src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/ECCurveType.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Security.Cryptography
+{
+    public enum ECCurveType : int
+    {
+        Unspecified = 0,
+        PrimeShortWeierstrass = 1,
+        PrimeTwistedEdwards = 2,
+        PrimeMontgomery = 3,
+        Characteristic2 = 4
```

Comment:
Nit: Please use the trailing comma, so that future edits don't show up as editing this line.